pyracf/common/security_admin.py

```diff
@@ -219,6 +211,15 @@ def _clear_state(self) -> None:
         self._segment_traits = {}
         self._trait_map = {}
 
+    def _reset_state(self) -> None:
+        """Reset state once requests are done."""
+        del self._segment_traits
+        del self._trait_map
+        del self.__preserved_segment_traits
+        self._segment_traits = {}
+        self._trait_map = {}
+        self.__preserved_segment_traits = {}
```

Comment:
It looks like we now have two different versions of this function in the form of `_clear_state` and `_reset_state` which accomplish similar things. I think that since all requests go through the `_make_request` function one at a time, we can just have one cleanup function call there. So, I think we can simplify things by just having the `_clear_state` function do everything. There are also some calls to `_clear_state` in some child classes, but I don't think this is necessary if we just make one call in `_make_request` to clean everything up at the end of the request, so we can change the name of the function to `__clear_state` to keep this private to `security_admin.py`.

```python
    def __clear_state(self) -> None:
        """Clear state for new request."""
        self._segment_traits = {}
        self._trait_map = {}
        self.__preserved_segment_traits = {}
        security_request.reset_request_xml()
```

Reply:
This conversation is still out standing.

Reply:
Looks good now.